### ISSUE TITLE
[patch #9416] AVR_TINY: Avoid conatraint "w" in delay.h

### ISSUE THREAD

**Issue by avrs-admin** [avrdudes/avr-libc#822]
Mon 24 Jul 2017 02:47:36 PM CEST

delay_basic.h might use constraint "w" for AVR_TINY which as an empty register class.

```
* include/util/delay_basic.h (_delay_loop_2) [__AVR_TINY__]: Avoid
constraint "w".

```

[file #41315: alibc-delay-tinyw.diff](https://savannah.nongnu.org/patch/download.php?file_id=41315)

This issue was migrated from https://savannah.nongnu.org/patch/?9416

**Comment by avrs-admin** [comment]
Georg-Johann Lay <gjlayde>
Wed 17 Jan 2018 04:58:44 PM CET

Ping #1

**Comment by sprintersb** [comment]
Fixed by integrating #902
